test/cypress/integration/24-event.createOrder.test.js

```diff
@@ -1,9 +1,17 @@
 describe('event.createOrder page', () => {
+  it("can't order if the event is over", () => {
+    cy.visit('/opensource/events/webpack-webinar');
+    cy.contains('Webinar: How Webpack Reached $400K+/year in Sponsorship & Crowdfunding');
+    cy.get('.cover .cta').should('not.exist');
+    cy.get('#tickets').should('not.exist');
+  });
+
   it('makes an order for a free ticket as an existing user', () => {
+    cy.clock(Date.parse('2017/10/01')); // Go back in time when the event was not over yet
```

Comment:
I've added a bunch of improvements to the PR, but the core of the fix for the tests is here: we mock the clock so the tests run as if it was 2017-10-01 (before the event was over). This way the tickets still appear on the page.

Reply:
Nice!